htsinfer/get_library_type.py

```diff
@@ -113,7 +113,7 @@ def _evaluate_mate_relationship(
             ids_2: As `ids_1` but for the putative second mate file.
         """
         self.results.relationship = StatesTypeRelationship.not_mates
-        if ids_1 == ids_2:
+        if ids_1 == ids_2 and len(ids_1) > 0 and len(ids_2) > 0:
```

Comment:
I guess you can just write `if ids_1 == ids_2 and ids_1 and ids_2:`, as a logical check will return `False` if a list is empty and `True` only if it has at least one item. And to make it a bit more efficient and easier to digest, I would change the order like so:

```py
if ids_1 and ids_2 and ids_1 == ids_2:
```

Reply:
Thanks, I changed that in the commit.